template/content/clients.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+var TChannel = require('tchannel');
+var HyperbahnClient = require('tchannel/hyperbahn');
+var fs = require('fs');
+var path = require('path');
+var Ringpop = require('ringpop');
+var Logtron = require('logtron');
+var Statsd = require('uber-statsd-client');
+var os = require('os');
+var process = require('process');
+var myLocalIp = require('my-local-ip');
+var Uncaught = require('uncaught-exception');
+
+var thriftFile = fs.readFileSync(
+    path.join(__dirname, 'thrift', 'service.thrift'), 'utf8'
+);
+
+var SERVICE_NAME = 'my-service';
+
+/* TODO:
+    - process reporter
+    - heap dump
+    - repl
+*/
+module.exports = ApplicationClients;
+
+function ApplicationClients(config, options) {
```

Comment:
I am working on a project with a monster number of clients. I will be decomposing it into a dependency graph by  exporting each client with a list of dependencies and then initializing everything with run-auto. This way I can keep client initialization code factored out into simple functions without needing to maintain a huge order-dependent initialization script.